hyperstone/hooks/context.py

```diff
@@ -1,10 +0,0 @@
-from typing import Any, Dict
-
-ONLY_HAS_GLOBALS = 1
```

Comment:
Who is supposed to change this global?
if the user is supposed to change it, maybe it should be a property of some class?